Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,1069 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' CONSTANTS '''
+
+removed_keys = ['endpoint', 'domain', 'hostname']
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def handle_error(self, message: dict = {}):
+        return_error(message['message'])
+
+    def domain_tags(self, hostname: str = None):
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}/tags',
+            ok_codes=[200, 403],
```

Comment:
you can also pass the ok_codes to the client object when initiating instead of passing this on each http_request. 
Why do we approve 403 code?

Reply:
The 403 is approved as it then returns the error message in a JSON dictionary. The handle_error then extracts the error message and displays it to the user. This is prettier than dumping the text of the call when anything other than 200 is received.